Cargo.toml

```diff
@@ -4,9 +4,10 @@ version = "0.8.0"
 authors = ["Simon Sapin <simon.sapin@exyr.org>"]
 license = "MIT"
 description = "(朽木) HTML/XML tree manipulation library"
-repository = "https://github.com/SimonSapin/kuchiki"
+repository = "https://github.com/kuchiki-rs/kuchiki"
 edition = "2018"
 
+
```

Comment:
also remove this space, I might have inadvertendly added it when resolving conflicts